<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportGenerator</class>
 <widget class="QWidget" name="ReportGenerator">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>503</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Report Generator</string>
  </property>
  <widget class="QTableWidget" name="reportTable">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>67</y>
     <width>520</width>
     <height>380</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>200</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>30</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Commercial Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>StartTime</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="exportButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>465</y>
     <width>110</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Export</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="fromDateEdit">
   <property name="geometry">
    <rect>
     <x>171</x>
     <y>21</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>M/d/yy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="toDateEdit">
   <property name="geometry">
    <rect>
     <x>291</x>
     <y>21</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_From">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>26</y>
     <width>150</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate Report From:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_To">
   <property name="geometry">
    <rect>
     <x>262</x>
     <y>26</y>
     <width>20</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>To:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="generateButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
